debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlJdbcContext.java

```diff
@@ -37,6 +37,9 @@
 
     protected static final String MYSQL_CONNECTION_URL = "jdbc:mysql://${hostname}:${port}/?useInformationSchema=true&nullCatalogMeansCurrent=false&useSSL=${useSSL}&useUnicode=true&characterEncoding=UTF-8&characterSetResults=UTF-8&zeroDateTimeBehavior=CONVERT_TO_NULL";
     protected static final String JDBC_PROPERTY_LEGACY_DATETIME = "useLegacyDatetimeCode";
+    protected static final String JDBC_PROPERTY_VERIFY_SERVER_CERT = "verifyServerCertificate";
```

Comment:
`verifyServerCertificate` is deprecedated for MySQL JDBC driver, you should look at value of `sslMode` config option.